src/tickets/entities/tickets.controller.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { Test, TestingModule } from '@nestjs/testing';
```

Comment:
move this file outside the entities folder

Reply:
done